src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -50,7 +50,7 @@ const ContinuousEmbeddingPage = ({ experimentId }) => {
   const [searchedGene, setSearchedGene] = useState();
 
   useEffect(() => {
-    dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
+    if (!config) dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
     dispatch(loadCellSets(experimentId));
```

Comment:
Add a check for the cellsets too?

Reply:
loadCellSets has an internal check which would not fire the request if cell set exists: https://github.com/hms-dbmi-cellenics/ui/blob/master/src/redux/actions/cellSets/loadCellSets.js#L13-L15 So we don't have to explicitly guard this